<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveLoginDialogClass</class>
 <widget class="QDialog" name="LiveLoginDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>LiveLoginDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LiveLoginDialog.qrc">
    <normaloff>:/LiveLoginDialog/res/live.ico</normaloff>:/LiveLoginDialog/res/live.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>450</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* font */
*
{
	font-family: &quot;Arial&quot;, &quot;Segoe UI&quot;, &quot;黑体&quot;, &quot;等线&quot;;
	font-size: 14px;
	font-weight: 400;
	line-height: 20px;
}
/* background */
QWidget#centralWidget
{
	background: rgba(36,41,46,0.9);
	background-position: top;
	background-image: url(:/LiveLoginDialog/res/login-background-600x180.png);
	background-repeat: no-repeat;
}

QLabel#applicationIconLabel
{
	border-image: url(:/LiveLoginDialog/res/pilipili-title-210x50.png);
}
/* avatar */
QLabel#avatarLabel
{
	border-image: url(:/LiveLoginDialog/res/avatar-mask.png);
	border-radius:60px;
	min-width:  120px;
    max-width:  120px;
    min-height: 120px;
    max-height: 120px;
    border-width: 0 0 0 0;
}

/* button */
.QPushButton, .QCheckBox, .QToolButton
{
	color: white;
	background-color: transparent;
}
/*RadioButton CheckButton*/
QRadioButton::indicator::unchecked {
	image: url(:/LiveLoginDialog/res/radiobutton-unchecked.png);
}
QRadioButton::indicator::checked { 
	image: url(:/LiveLoginDialog/res/radiobutton-checked.png);
}
QCheckBox::indicator { 
	width: 15px;
	height: 30px;
}
QCheckBox::indicator::unchecked {	
	image: url(:/LiveLoginDialog/res/checkbox_unchecked.png);
}
QCheckBox::indicator::checked { 
	image: url(:/LiveLoginDialog/res/checkbox_checked.png);
}

.QCheckBox::indicator { 
	width: 15px;
	height: 15px;
}
.QCheckBox::indicator::unchecked {	
	image: url(:/LiveLoginDialog/res/radiobutton-unchecked.png);
}
.QCheckBox::indicator::checked { 
	image: url(:/LiveLoginDialog/res/radiobutton-checked.png);
}
.QPushButton:hover, .QCheckBox:hover
{
	color: yellow;
}
.QPushButton:pressed, .QCheckBox:pressed
{
	color: #FFCC66;
}
.QPushButton:focus, .QCheckBox:focus
{
	outline: none
}
/* login button */
QPushButton#loginPushButton {
  background: #0078d7;
  border: 2px solid rgba(0,0,0,0.4)
}
QPushButton#loginPushButton:hover {
  background: #1CA8FA;
}
QPushButton#loginPushButton:pressed {
  background: #2A67FF;
}
QPushButton#loginPushButton:disabled {
  background: #666;
}
/* account password */
QLineEdit#passwordLineEdit, QLineEdit#accountLineEdit
{
	border: 2px solid rgba(27,128,255,0.5);
	background: rgba(45,64,72,0.8);
	color: white;
    selection-background-color:green;
}
QLineEdit#passwordLineEdit:focus, QLineEdit#accountLineEdit:focus
{
	background: rgba(80,100,110,0.8);
}
/* tooltip */
QToolTip
{
	color: yellow;
	border: 2px solid #333;
	background-color: black;
}</string>
   </property>
   <layout class="QVBoxLayout" name="centralLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="titlebarWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="applicationIconLabel">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>50</height>
          </size>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="titlebarHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>303</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="titleMenuVerticalLayoutWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="titleMenuVerticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="titleMenuWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>32</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="titleMenuHorizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="settingToolButton">
               <property name="minimumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="icon">
                <iconset resource="LiveLoginDialog.qrc">
                 <normaloff>:/LiveLoginDialog/res/setting.png</normaloff>:/LiveLoginDialog/res/setting.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="minimizeToolButton">
               <property name="minimumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>最小化</string>
               </property>
               <property name="icon">
                <iconset resource="LiveLoginDialog.qrc">
                 <normaloff>:/LiveLoginDialog/res/minus-square.png</normaloff>:/LiveLoginDialog/res/minus-square.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="closeToolButton">
               <property name="minimumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>关闭</string>
               </property>
               <property name="icon">
                <iconset resource="LiveLoginDialog.qrc">
                 <normaloff>:/LiveLoginDialog/res/close-square.png</normaloff>:/LiveLoginDialog/res/close-square.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="mainVerticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>6</width>
        <height>66</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="avatarBox">
      <item>
       <spacer name="avatarLeftHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="avatarLabel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="avatarRightHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="formHorizontalLayout">
      <item>
       <spacer name="formLeftHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="formContextWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>122</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>122</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="accountHorizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="accountLabel">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="LiveLoginDialog.qrc">:/LiveLoginDialog/res/account.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="accountLineEdit">
             <property name="minimumSize">
              <size>
               <width>280</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolTip">
              <string>账号</string>
             </property>
             <property name="placeholderText">
              <string>Account here.</string>
             </property>
             <property name="editable" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="passwordHorizontalLayout">
           <item>
            <widget class="QLabel" name="passwordLabel">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="LiveLoginDialog.qrc">:/LiveLoginDialog/res/password.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordLineEdit">
             <property name="minimumSize">
              <size>
               <width>280</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolTip">
              <string>密码</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Password here.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="buttonSheetHorizontalLayout">
           <item>
            <widget class="QCheckBox" name="rememberPasswordCheckBox">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>记住密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoLoginCheckBox">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>自动登录</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="forgetPasswordPushButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>找回密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="registerPushButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>注册账号</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="formRightHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="loginHorizontalLayout">
      <item>
       <spacer name="loginLeftHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loginPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="loginRightHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginPushButton</tabstop>
  <tabstop>accountLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>rememberPasswordCheckBox</tabstop>
  <tabstop>autoLoginCheckBox</tabstop>
  <tabstop>forgetPasswordPushButton</tabstop>
  <tabstop>registerPushButton</tabstop>
  <tabstop>minimizeToolButton</tabstop>
  <tabstop>closeToolButton</tabstop>
 </tabstops>
 <resources>
  <include location="LiveLoginDialog.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>false</bool>
  </property>
 </designerdata>
</ui>
